src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -175,21 +258,39 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
   }
 }
 
+PipelineRuns.defaultProps = {
+  filters: []
+};
+
+export function fetchFilters(searchQuery) {
+  const queryParams = new URLSearchParams(searchQuery);
```

Comment:
Query params always defined ? Why would we have multiple query params with comma seperation ?

Reply:
`props.search.location` is '' by default so it'll be always defined, if there are no params in the query then it'll just iterate over nothing and return no filters